server/internal/heavy/components.go

```diff
@@ -293,6 +302,25 @@ func newComponents(ctx context.Context, cfg configuration.Configuration, genesis
 		}
 	}
 
+	// Exporter
+	var (
+		recordExporter *exporter.RecordServer
+	)
+	{
+		recordExporter = exporter.NewRecordServer(Pulses, RecordPosition, Records, JetKeeper)
+		lis, err := net.Listen("tcp", cfg.Exporter.Addr)
+		if err != nil {
+			return nil, errors.Wrap(err, "failed to open port for Exporter")
+		}
+		grpcServer := grpc.NewServer()
```

Comment:
is it possible to stop it?
since when we run tests with 'count' it will live for all this period and consume resources

Reply:
It should work, because it's a part of a heavy node